### PR TITLE
Update package to google-genai in `examples/Classify_text_with_embeddings.ipynb`

### PR DESCRIPTION
**Pull request by Jenniebn**
Update package to google-genai in `examples/Classify_text_with_embeddings.ipynb`

* Updated `google-generativeai` to use `google-genai` package instead.
* Updated outputs and tested locally.
* Refactored imports
* Process 100 rows in 1 batch request for getting embeddings. (100 is the max limit `BatchEmbedContentsRequest.requests: at most 100 requests can be in one batch`) .The original issue didn't require this optimization, but it was needed since free gemini API credits are insufficient for 600 requests. Now, it can be done in 6 requests.
* Replaced `Sigmoid` with `Softmax` for multi-class classification and set `from_logits=False`  
* Formatted with nbfmt and nblint tool

### CODE REVIEW

File: examples/Classify_text_with_embeddings.ipynb [hunk 1]
```diff
@@ -6,12 +6,12 @@
         "id": "Tce3stUlHN0L"
```
[the remark this answers is not in the document]
Comment (Giom-V) [reply examples/Classify_text_with_embeddings.ipynb:6]: I did some test, don't use the new model by default as the quota is too limited.

**Comment by Jenniebn** [comment]
> Thanks for the submission @Jenniebn! The code is good and well formatted.
> 
> I just added a few comments so that we improve the explanations while we're at it.

Thank you for your review. I just implemented everything you commented! Feel free to let me know if there is anything else that needs to be updated!

**Review by Giom-V — APPROVED** [review]

**Comment by Giom-V** [comment]
Thanks for the submission @Jenniebn. It's approved and merged!
